src/core/server/bootstrap.ts

```diff
@@ -55,6 +55,11 @@ export async function bootstrap({
     onRootShutdown('Kibana REPL mode can only be run in development mode.');
   }
 
+  if (cliArgs.optimize) {
+    // --optimize is deprecated and does nothing now, avoid starting up and just shutdown
```

Comment:
I tried, but the logger is in buffering mode here and I don't think we should write something to stdout or something without considering the other flags... I figured the `--help` and release notes are sufficient to warn people that they don't need to call this. In 8.0 we'll remove the flag and then it will stop working, I don't think we need to figure out how to make it log. https://github.com/elastic/kibana/pull/73154#discussion_r468330149